release/ray_release/scripts/ray_bisect.py

```diff
@@ -3,7 +3,7 @@
 import os
 import json
 import time
-from typing import List
+from typing import List, Dict
```

Comment:
nit: let's keep these alphabetically sorted

```suggestion
from typing import Dict, List
```